extensions/ql-vscode/src/view/model-editor/MethodRow.tsx

```diff
@@ -218,3 +246,14 @@ function sendJumpToUsageMessage(method: Method) {
     usage: method.usages[0],
   });
 }
+
+function forEachModeledMethod(
```

Comment:
I'm not totally happy with this but it's the cleanest thing I could come up with right now. The idea is that when there are no modeled methods we still need to render one set of boxes, but passing `undefined` as the modeled method.